src/v8/v8.cc

```diff
@@ -586,11 +586,17 @@ void V8::getModuleFunctionImpl(std::string_view function_name,
     *function = nullptr;
     return;
   }
+
```

Comment:
Nit: could you remove this empty line? Otherwise it exist in one variant, but not the other.

Reply:
resolved. Thanks for the comment!